### PR TITLE
Replace bootstrap by provision only mode, lower score requirements

### PR DESCRIPTION
**Pull request by vdombrovski**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -36,8 +36,8 @@ openio_replicas: 3
 # this `.{}` will be filled by `format_location()`
 openio_location: "{{ ansible_hostname }}.{}"
 
-openio_minimal_score_for_volume_admin_bootstrap: 50
-openio_minimal_score_for_directory_bootstrap: 50
+openio_minimal_score_for_volume_admin_bootstrap: 10
```
Comment (cdelgehier) [inline defaults/main.yml]: Why ?

File: tasks/main.yml [hunk 2]
```diff
@@ -24,6 +24,7 @@
     owner: root
     group: root
     mode: 0644
+    backup: true
```
Comment (cdelgehier) [inline tasks/main.yml]: Can you explain ?

File: defaults/main.yml (hunk 1 above)
Comment (vdombrovski) [reply defaults/main.yml]: Not really related, but in some cases scores take too long to reach 50, so lowering them is a good idea.

File: tasks/main.yml (hunk 2 above)
Comment (vdombrovski) [reply tasks/main.yml]: Making backups of newly provisioned files is the way to go. Again not related to PR, but useful addition imo.

File: defaults/main.yml (hunk 1 above)
Comment (cdelgehier) [reply defaults/main.yml]: @racciari @vincent-legoll can you validate this change ?

**Pull request merged**